app/src/ui/history/compare.tsx

```diff
@@ -33,6 +33,8 @@ interface ICompareSidebarProps {
   readonly dispatcher: Dispatcher
   readonly currentBranch: Branch | null
   readonly sidebarHasFocusWithin: boolean
+  /**Determines whether or not to focus compare branch selector */
+  readonly focusBranchSelector: boolean
```

Comment:
Can we use a name here that's prefixed with `initial` to indicate that the value is only used when initializing the component. Something like `initialShowBranchList`?

Reply:
I'll use that until I can think of something better...